modules/core/packet-server/keeper/keeper.go

```diff
@@ -59,7 +59,7 @@ func (k Keeper) SendPacket(
 	// Lookup counterparty associated with our source channel to retrieve the destination channel
 	counterparty, ok := k.ClientKeeper.GetCounterparty(ctx, sourceChannel)
 	if !ok {
-		return 0, channeltypes.ErrChannelNotFound
+		return 0, errorsmod.Wrap(clienttypes.ErrCounterpartyNotFound, sourceChannel)
```

Comment:
We can add a more descriptive error string here when we decide what to do with channel aliasing. For now I kept it simple.